packages/test/test-end-to-end-tests/src/test/summaryDataStoreStats.spec.ts.ts

```diff
@@ -94,13 +93,11 @@ describeNoCompat("Generate Summary Stats", (getTestObjectProvider) => {
         }
     }
 
-    const createContainer = async (): Promise<IContainer> => provider.createContainer(runtimeFactory);
-
-    let mockLogger: MockLogger;
+    const mockLogger = new MockLogger();
+    const options: ITestLoaderOptions = { logger: mockLogger };
```

Comment:
nit: No need to declare a variable here. you can directly pass `{ logger: mockLogger }` to createContainer.

Reply:
i think i still need to declare once , otherwise the getGenerateSummaryEvent function can't access it?